HealthcareSystem/GraphicalEditor/Repository/MapObjectRepository.cs

```diff
@@ -0,0 +1,59 @@
+﻿using GraphicalEditor.Models;
+using GraphicalEditor.Repository.Intefrace;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GraphicalEditor.Repository
+{
+    public class MapObjectRepository : IMapObjectRepository
```

Comment:
What is the point of having a map repository, and a separate map object repository? They both keep a list of map objects (the entire map, I assume). What are their responsibilities and how do they differ from eachother?

Reply:
i am not sure i understand the  question, another one is interface

Reply:
I meant the FileRepostory class, and the MapObjectRepository class. What is the point of having both of them? They are both just repositories of MapObjects (and MapObjectRepository even used the FileRepository). What are the exact responsibilities of those classes?

Reply:
MapObjectRepository and its interface are deleted